### ISSUE TITLE
download stalling because congestion limited

### ISSUE THREAD

**Issue by marten-seemann** [lucas-clemente/quic-go#343]
When running the Chrome small file download integration test, about every 1 in 100 attempts the download stalls on Ubuntu, because we're congestion limited.

See attached [log.txt](https://github.com/lucas-clemente/quic-go/files/566188/log.txt).

**Comment by marten-seemann** [comment]
I ran a couple of tests, and this is what I found out: The download begins to stall once we start queueing RTOs. This is a local, lossless connection, so that isn't expected to happen at all.

Queueing of RTOs begins once the client sends an ACK with a very high delay time. Usually, the ACK frames received in the tests have delay times of about 10 μs, the ACK frame that triggers the RTOs has a delay time of 23 ms (!). The RTT estimate at that point was 5 ms.
